modules/web/src/app/core/interceptors/error-notifications.ts

```diff
@@ -44,7 +44,11 @@ export class ErrorNotificationsInterceptor implements HttpInterceptor {
   ];
 
   // Array of endpoints that should be silenced in the UI.
-  private readonly _silencedEndpoints = ['providers/gke/validatecredentials', 'presets?name='];
+  private readonly _silencedEndpoints = [
+    'providers/gke/validatecredentials',
+    'presets?name=',
+    'CLUSTER_AUTOSCALING_APP_DEF_NAME',
```

Comment:
Do we want cluster autoscaler app name here or the variable name? Also, the endpoint should be a bit more complete rather than just the app name.